appveyor.ps1

```diff
@@ -26,6 +37,30 @@ Get-ChildItem $ReferenceDocset -Directory -Exclude 'docs-conceptual','mapping' |
         $MamlOutputFolder = Join-Path "$PSScriptRoot\maml" "$Version\$ModuleName"
         $CabOutputFolder = Join-Path "$PSScriptRoot\updatablehelp" "$Version\$ModuleName"
 
+        # Process the about topics if any
+        $AboutFolder = Join-Path $ModulePath "About"
+
+        if(Test-Path $AboutFolder)
+        {
+            Get-ChildItem "$aboutfolder/about_*.md" | ForEach-Object {
+                $aboutFileFullName = $_.FullName
+                $aboutFileOutputName = "$($_.BaseName).help.txt"
+                $aboutFileOutputFullName = Join-Path $MamlOutputFolder $aboutFileOutputName
+
+                $pandocArgs = @(
+                    "--from=markdown",
+                    "--to=plain+multiline_tables+inline_code_attributes",
+                    "--columns=75",
+                    "--output=$aboutFileOutputFullName",
+                    "--ascii",
```

Comment:
According to this, this isn't necessary as it's only for HTML: https://pandoc.org/MANUAL.html

Also, if we could take the opportunity to use UTF-8 (BOM-less) on these, given that we've moved PowerShell to that all up, that'd be awesome. Don't know what the default is on Pandoc, but I'd guess it might already be....

Reply:
I chose ASCII because of problems we have had with BOM-less UTF-8 in the docs tool chain. However, I did just test this with a sample that uses special characters and I can verify that it outputs UTF-8 (BOM-less) text and not plain ASCII. So we can remove that parameter.

Reply:
Removed `--ascii`. Thanks.